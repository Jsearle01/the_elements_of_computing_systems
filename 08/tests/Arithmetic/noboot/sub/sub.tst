load sub.asm,
output-file sub.out,
compare-to sub.cmp,
output-list
    D%D1.6.1
    RAM[0]%D1.6.1
    RAM[256]%D1.6.1
    RAM[257]%D1.6.1
    RAM[258]%D1.6.1;

set RAM[0] 258,
set RAM[256] 5,
set RAM[257] 10,

repeat 100 {
  ticktock;
}

output;

