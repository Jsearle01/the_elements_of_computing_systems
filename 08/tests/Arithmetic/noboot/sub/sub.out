|   D    | RAM[0] |RAM[256]|RAM[257]|RAM[258]|
|     10 |    257 |     -5 |     10 |      0 |
